src/providers/completion/controllerCompletionItemProvider.ts

```diff
@@ -215,6 +218,65 @@ export class ControllerCompletionItemProvider implements CompletionItemProvider
 		return completions;
 	}
 
+	/**
+	 * Get Alloy API completions
+	 *
+	 * @param {String} linePrefix line prefix text
+	 *
+	 * @returns {Array}
+	 */
+	public getAlloyApiCompletions (linePrefix: string) {
+		const { types } = this.completions.alloy;
+		const matches = linePrefix.match(/(Alloy\.(?:(?:[A-Z]\w*)\.?)*)([a-z]\w*)*$/);
```

Comment:
We allow coming into here with `alloy.`, but it looks like we're not allowing the lower case here. Based off the usage for Titanium API completions I think we only want to allow `Alloy`?